app/models/application_criterion.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class ApplicationCriterion < ApplicationRecord
+  self.table_name = 'application_criteria'
+  # == Attributes ===========================================================
+  # == Callbacks ============================================================
+  # == Constants ============================================================
+  # == Extensions ===========================================================
+  # serialize :frequent_asked_info, coder: HashSerializer
+  # store_accessor :attr1, :attr2, :attr3, :attr4, :attr5
+  # == Relationships ========================================================
+  belongs_to :job
+  # == Scopes ===============================================================
+  # == Validations ==========================================================
+  # == Instance Methods =====================================================
+  # == Class Methods ========================================================
+  def questions
+    # hh = []
+    # hh << form_structure.map { |section| section }
```

Comment:
what are the hh's doing here?

Reply:
That commented section should be removed.